R/conda.R

```diff
@@ -1106,9 +1106,26 @@ get_python_conda_info <- function(python) {
     else
       file.path(root, "bin/conda")
   } else {
-    # not base env, parse conda-meta history to find the conda binary
-    # that created it
-    conda <- python_info_condaenv_find(root)
+    # not base env
+    conda <- find_conda()
+    if (is.null(conda)) {
+      if (is_windows()) {
+        # in Anaconda base env, conda.exe lives under Scripts
+        exe <- file.path(root, "Scripts", "conda.exe")
+        if (file.exists(exe))
+          conda <- exe
```

Comment:
Not sure if this is still necessary with the above handling for base conda envs

```
  if (dir.exists(file.path(root, "condabin"))) {
    # base conda env
    conda <- if (is_windows())
      file.path(root, "condabin/conda.bat")
    else
      file.path(root, "bin/conda")
  } 
```

Reply:
https://github.com/rstudio/reticulate/blob/14d732716c006d72500be1db57388645c3800183/R/conda.R#L553-L560

`conda.bat` replaces `conda.exe` whenever `conda_binary(".../conda.exe")` is used too